services/authorizerService.js

```diff
@@ -0,0 +1,31 @@
+/*
+Authorizer services
+*/
+import Authorizer from "../models/authorizerModel.js";
+
+const getRequestStatusId = async (id) => {
+    try {
+        const idRequest = await Authorizer.getStatusId(id);
+        console.log("ID a comprobar:", idRequest);
+        if (!idRequest) {
+            return null;
+        }
+        else {
+            if (idRequest == 1)
+                return 2;
+            else if (idRequest == 2)
+                return 3;
```

Comment:
Discussed this with @BassedWarrior @a01029143 and the status should be changed to 4 (cotización del viaje en el nuevo orden autorización de Request_status), regardless of who authorizes it (N1, N2), checking for status 2 or 3 instead of 1, since status 1 will be used for drafts of requests.

```suggestion
        else {
            if (idRequest == 2 || idRequest == 3)
                return 4;
```